tests/containers/tang/Containerfile

```
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora-minimal:40
+FROM registry.fedoraproject.org/fedora-minimal:41
 
 RUN microdnf -y install tang && microdnf clean all && rm -rf /var/cache/yum
```

any use of `microdnf` now should just be updated to `dnf` since `dnf` isn't python based any longer.

```suggestion
RUN dnf -y install tang && dnf clean all && rm -rf /var/cache/yum
```